src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -63,7 +63,7 @@
         "symfony/proxy-manager-bridge": "<4.4.19",
         "symfony/security-bundle": "<5",
         "symfony/security-core": "<5.3",
-        "symfony/validator": "<5.4.25|>=6,<6.2.12|>=6.3,<6.3.1"
```

Comment:
Not sure if it would better to add a conflict to `symfony/validator` `7.0` as adding a conflict after a release in this case (5.4 and 6.3) will composer maybe just install a old version of the `doctrine-bridge` and then still run into the same error 🤔.

But it feels strange to add a conflict to `symfony/validator` as it has not even a optional dependency to the bridge.

Reply:
I think we should add this conflict rule here (where it is legitimate), but we should also add it in validator 7.0 to avoid the issue of installing validator 7.0 and an outdated doctrine-bridge.

Reply:
That would work I think. @nicolas-grekas what do you think about that?